test/test_ops.py

```diff
@@ -37,12 +37,10 @@ def test_unsupported_dtypes(self, device, dtype, op):
         # sample_inputs can have a function for generating the input that doesn't work for specified dtype
         # https://github.com/pytorch/pytorch/issues/49024
         with self.assertRaises(RuntimeError):
-            samples = op.sample_inputs(device, dtype)
-            if len(samples) == 0:
-                self.skipTest("Skipped! No sample inputs!")
-
             # NOTE: only tests on first sample
-            sample = samples[0]
+            sample = op.get_one_sample_input(device, dtype)
+            if sample is None:
```

Comment:
We should probably just remove this check: I think every current OpInfo has at least one SampleInput and every new OpInfo should have at least one